ledfx/integrations/mqtt_hass.py

```
@@ -11,10 +11,23 @@
 from ledfx.effects.audio import AudioInputSource
 from ledfx.events import Event
 from ledfx.integrations import Integration
+from ledfx.utils import generate_defaults, inject_missing_default_keys
```

Ruff (0.8.2)

`ledfx.utils.generate_defaults` imported but unused

Remove unused import

(F401)